src/pages/studies/SearchBar.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react';
+import { VisualType } from '@ferlab/ui/core/components/filters/types';
+import { ISqonGroupFilter } from '@ferlab/ui/core/components/QueryBuilder/types';
+import { Select, Tag } from 'antd';
+
+import history from 'services/history';
+
+import { ItemProps } from './SidebarFilters';
+import { updateFilters } from './utils';
+
+import styles from './SearchBar.module.scss';
+
+type OwnProps = {
+  options: ItemProps[];
+  filters: ISqonGroupFilter;
+};
+
+const extractCodesFromFilter = (filters: ISqonGroupFilter) => {
+  const find = filters.content.find(
+    (s: { content: { field: string } }) => s.content.field === 'code',
+  );
+  return find ? find.content.value : [];
+};
+
+const SearchBar = ({ options, filters }: OwnProps) => {
+  const [selected, setSelected] = React.useState<string[]>([]);
+
+  const selectedStudyCodes = extractCodesFromFilter(filters);
+
+  useEffect(() => {
+    const updateSelected = options.filter((f) =>
```

Comment:
why not creating a function that takes the options and convert id to and array? 
something like:
const optionsListified = yourFunc(rawOptions)
setSelected(optionsListified)